lib/trento/discovery/payloads/cluster/cluster_discovery_payload.ex

```diff
@@ -46,8 +46,7 @@ defmodule Trento.Discovery.Payloads.Cluster.ClusterDiscoveryPayload do
     end
   end
 
-  defp offline?(%{"id" => _} = payload) when map_size(payload) == 1, do: true
-  defp offline?(%{"id" => _, "name" => _} = payload) when map_size(payload) == 2, do: true
+  defp offline?(%{"online" => false}), do: true
```

Comment:
nitpick:
maybe for the next time. We don't need to have `if/else` above. We can use the power of pattern matching:
```
def changeset(cluster, %{"online" => false} = attrs), do:
    changeset_offline(cluster, attrs)

def changeset(cluster, attrs), do:
    changeset_online(cluster, attrs)
```